src/ui/Heading/Heading.stories.js

```diff
@@ -0,0 +1,30 @@
+import Heading from "./Heading.svelte";
+
+export default {
+  title: "Example/Heading",
```

Comment:
Same question as above -- do these `title` and `text` props show up in the app somewhere?